ocs_ci/ocs/ripsaw.py

```diff
@@ -101,13 +102,24 @@ def get_uuid(self, benchmark):
             str: the UUID of the test
 
         """
-        output = self.pod_obj.exec_oc_cmd(f'exec {benchmark} -- env')
-        uuid = ''
-        for line in output.split():
-            if 'uuid=' in line:
-                uuid = line.split('=')[1]
+        count = 0
+        while count <= 5:
```

Comment:
why using while loop instead of for loop?

Reply:
no particular reason